package.json

```diff
@@ -47,15 +47,15 @@
   },
   "devDependencies": {
     "ava": "^0.17.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.1.0",
     "eslint-ava-rule-tester": "^0.1.1",
     "inject-in-tag": "^1.1.1",
     "nyc": "^15.1.0",
     "pify": "^5.0.0",
     "xo": "^0.44.0"
   },
   "peerDependencies": {
-    "eslint": "^7.32.0"
+    "eslint": "^8.1.0"
```

Comment:
`8.x` ? cc @gao-sun

Reply:
`eslint-plugin-sql` requires `^8.1.0`, let's align with that